.travis/osx_openresty_runner.sh

```diff
@@ -33,6 +33,26 @@ before_install() {
     sudo cpanm --notest Test::Nginx >build.log 2>&1 || (cat build.log && exit 1)
     export_or_prefix
     luarocks install --lua-dir=${OPENRESTY_PREFIX}/luajit luacov-coveralls --local --tree=deps
+
+    # spin up kafka cluster for tests (1 zookeper and 1 kafka instance)
```

Comment:
we do not need to install the `zookeeper`, `kafka`.
only run the mini test cases(t/admin/*.t) for OSX system.

Reply:
oh okays. I didnt know that

Reply:
@membphis done.